webapi/src/test/scala/org/knora/webapi/E2ESpec.scala

```diff
@@ -213,3 +213,17 @@ class E2ESpec(_system: ActorSystem)
       }
     }
 }
+
```

Comment:
This looks like something for the `org.knora.webapi.messages` package. Either under `main` or `test`, depending where it is used.

Reply:
to be honest, I just copied this from `ITKnoraLiveSpec.scala` because so far this has been the only test type with SIPI running. Do you want me to move it to messages in both places? Or can I leave it for now?

Reply:
done